src/components/page/video.json

```diff
@@ -0,0 +1,18 @@
+{
+  "collectionName": "components_page_videos",
+  "info": {
+    "displayName": "Video",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "title": {
+      "required": true,
+      "type": "string"
+    },
+    "url": {
+      "required": true,
+      "type": "string"
+    }
```

Comment:
My only concern is communicating that this can only contain a youtube string. Maybe we can just add a little text description to this field? 🤔

Reply:
[it seems the text description is actually added via the UI](https://docs.strapi.io/user-docs/content-manager/configuring-view-of-content-type#edit-view-display)